easybuild/easyblocks/m/mvapich2.py

```diff
@@ -69,14 +67,10 @@ def configure_step(self):
         add_configopts.append('--with-rdma=%s' % self.cfg['rdma_type'])
 
         # enable specific support options (if desired)
-        if self.cfg['withmpe']:
-            add_configopts.append('--enable-mpe')
```

Comment:
how about issuing a warning that this is deprecated?

do we know when this is going to be removed?